src/runtime/Loader.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+
+interface Props {
+  render: JSX.Element | undefined;
+}
+export const LoaderContext = React.createContext(undefined);
```

Comment:
Don't export this. Make it a private static member of the Loader class. This is how Component & Protocol do it currently.

`private static _RenderContext = React.createContext({ });`